tcms/testruns/migrations/0012_auto_20210217_0446.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.1.6 on 2021-02-17 04:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('testruns', '0011_add_planning_fields'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='historicaltestexecution',
+            name='close_date',
+        ),
+        migrations.RemoveField(
+            model_name='testexecution',
+            name='close_date',
+        ),
```

Comment:
This will break existing installations and destroy information. See `migrations.RenameField`